moby/moby
# add selinux policy for centos-7

**- What I did**

Added selinux policy for centos-7. Fixes issue #28406.

**- How I did it**

Sync with latest selinux policy for RHEL7 which is in RHEL RPM `docker-selinux-1.10.3-57.el7.x86_64` because that policy works on RHEL7.3.

I downloaded the SRPM on an Amazon EC2 instance of RHEL7:
```
$ sudo yumdownloader --enablerepo='*' --source docker-1.10.3-57.el7
```

And then cracked it open to get to the `docker.spec` file:
```
$ rpm2cpio docker-1.10.3-57.el7.src.rpm | cpio -idmv
```

Found the selinux policy is using lsm5/container-selinux@583a67ffdf9eef9afc233ace0f841d5eeef28fb3.

I use PR #25334 for reference in constructing this PR.

**- How to verify it**

Build a docker-engine-selinux centos-7 RPM package and install on RHEL7.3 with selinux enabled:
```
$ make DOCKER_BUILD_PKGS=centos-7 rpm
```

Should also make sure this works on CentOS7.2 with selinux enabled because CentOS7.3 is not released yet.

**- Description for the changelog**


Update selinux policy for distros based on RHEL7.3.

**- A picture of a cute animal (not mandatory but encouraged)**

🐰 

Signed-off-by: Andrew Hsu <andrewhsu@docker.com>
